deluge/ui/console/modes/connectionmanager.py

```diff
@@ -102,6 +102,7 @@ def _do_add(self, result, **kwargs):
         if not result or kwargs.get('close', False):
             self.pop_popup()
         else:
+            assert isinstance(result['port']['value'], int)
```

Comment:
Can you remove this change please

Reply:
Sure! Must have slipped from the other branch

Reply:
Excellent thanks, I'll merge later today

Do you know how to rebase to single commit and update the commit message? We would like to always have a summary of why and what changed in the commit message body, along with any ticket references. 

If you can give it a go that would be great, otherwise is not a big issue on this one.

Helpful commit message tips: https://cbea.ms/git-commit/#seven-rules

Reply:
By rebase to single commit do you mean "squash"? Yes, I can do that.

Reply:
It seems I did it. There is one commit that encapsulates all of the changes, but the previous commits are also there... The message of the squashed commit is updated to contain the context, as well as the link to the ticket.

Reply:
assuming you have the base repo(https://github.com/deluge-torrent/deluge) setup in your git clone as deluge-torrent. if you run in terminal from the deluge folder: `git rebase deluge-torrent/develop` that should update the repo and put your commit last. Currently it looks like its out of date but then commits merged in.(why there is 12 commits to the PR vs 1)

Reply:
cas could always just cherry-pick your change as well but if you clean it up its a little easier.as he said `If you can give it a go that would be great, otherwise is not a big issue on this one.`

Reply:
just thought id try to help on how to do the rebase.

Reply:
I see... I was somehow under impression that I should rebase against my fork of the deluge instead of the original one and that is what I did. Still not sure how I ended up with 12 commits though... I was pretty clear to squash the commits. Thanks for your help @doadin! I will keep that in mind

Reply:
np, yea you want to rebase against the official deluge git repo. Rebasing updates with all the commits at the time of the rebase then applies your changes after. So thats why it should go down to one, because after updating your commit should be the only different one.